sql/2019/17_CDN/17_05.sql

```diff
@@ -0,0 +1,9 @@
+#standardSQL
+# 17_05: Percentage of responses with strict-transport-security header
+SELECT
+  _TABLE_SUFFIX AS client,
+  ROUND(SUM(IF(LOWER(respOtherHeaders) LIKE "%strict-transport-security%", 1, 0)) / COUNT(0), 4) AS pctStrictTransportSecurity

```

Comment:
I also find it helpful to see the numerator and denominator split out into their own fields to get a sense of scale. Could you add each of these:

```sql
COUNTIF(LOWER(respOtherHeaders) LIKE "%strict-transport-security%") AS freq,
COUNT(0) AS total,
ROUND(COUNTIF(LOWER(respOtherHeaders) LIKE "%strict-transport-security%") * 100 / COUNT(0), 2) AS pct
```

Reply:
These suggestions apply throughout the CDN metrics

Reply:
will make the changes

Reply:
Changed 17_5, 6, 7, 8, 9, 10, 11 to (a) return freq, total, pct (b) use COUNTIF  in place of  SUM(IF()).
Results attached in following sheet
https://docs.google.com/spreadsheets/d/1EPa--z9DVQBh_rLpw5lqZNAcMooKKloXibyRtAUVtzk/edit?usp=sharing